### ISSUE TITLE
Great Instructions in your readme

### ISSUE THREAD

**Issue by m4v15** [FACK1/fid-or#16]
Well done for being the only group to actually give instructions on how to run your code (even though they are a little bit wrong 🙃 )  eg https://github.com/FACK1/mo-raz/issues/31, https://github.com/FACK1/had-am-sa/issues/40

**Comment by orjwan98** [comment]
Yeah it's been brought to our attention that we messed up a few, hehe. But we're working on them :)

**Comment by jema28** [comment]
I agree, really great step by step local setup instructions, makes it very quick and easy to set up the project! 💯

**Comment by jema28** [comment]
@orjwan98 @fdo2 Just one thing missing: `npm run dev` before going to your local host (step 9).
